parm/config/config.base.emc.dyn

```diff
@@ -251,39 +252,40 @@ case "${APP}" in
     export DO_COUPLED="YES"
     export DO_WAVE="YES"
     export WAVE_CDUMP="both"
-    export nems_temp="leapfrog_atm_wav"
+    export confignamevarfornems="leapfrog_atm_wav"
     ;;
   S2S)
     export DO_COUPLED="YES"
     export DO_OCN="YES"
     export DO_ICE="YES"
-    export CCPP_SUITES="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
-    export nems_temp="cpld"
+    export CCPP_SUITE="FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+    export confignamevarfornems="cpld"
     ;;
   S2SW)
     export DO_COUPLED="YES"
     export DO_WAVE="YES"
     export DO_OCN="YES"
     export DO_ICE="YES"
-    export CCPP_SUITES="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+    export CCPP_SUITE="FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
     export WAVE_CDUMP="both"
-    export nems_temp="cpld_wave"
+    export confignamevarfornems="cpld_wave"
+    export cplwav2atm=".true."
     ;;
   ATMAERO)
     export DO_COUPLED="YES"
     export DO_AERO="YES"
-    export nems_temp="atm_aer"
+    export confignamevarfornems="atm_aer"
     ;;
   *)
     echo "Unrecognized APP: ${1}"
     exit 1
     ;;
 esac
 
-if [[ "$SDATE" -lt "2019020100" ]]; then # no rtofs in GDA
-  export DO_WAVE="NO"
-  echo "WARNING: Wave suite turned off due to lack of RTOFS in GDA for SDATE"
-fi
+# if [[ "$SDATE" -lt "2019020100" ]]; then # no rtofs in GDA
```

Comment:
This could be turned back on and an extra "if APP+ATMW" could be added.

Reply:
Will this be turned back on before merging by adding the If APP=ATMW conditional?

Reply:
TBD when we fix ATMW.